docs/docs-components/COMPONENT_DATA.js

```diff
@@ -1776,6 +1776,55 @@ const GENERAL_COMPONENT_LIST: $ReadOnlyArray<ListItemType> = [
       figma: null,
     },
   },
+  {
+    svg: <Popover />,
+    name: 'PopoverEducational',
+    description:
+      'PopoverEducational is a floating container that introduces users to elements on the screen. Used for education or onboarding experiences. ',
```

Comment:
nit: there's an extra whitespace at the end of the string

Reply:
DONE